File: 01_Python/07_forLoop/forLoop.py
"""
-----------------------------------------
Topic : For Loop
Language : Python
Bootcamp : Data Analytics Bootcamp
Author : Shubham Waghmare
-----------------------------------------
"""

# ==========================
# Basic for Loop
# ==========================

for i in range(5):
    print(i)

# ==========================
# Custom Range
# ==========================

for i in range(1, 6):
    print(i)

# ==========================
# Step Value
# ==========================

for i in range(0, 11, 2):
    print(i)

# ==========================
# Reverse Loop
# ==========================

for i in range(10, 0, -1):
    print(i)

# ==========================
# Loop Through String
# ==========================

name = "Shubham"

for char in name:
    print(char)

# ==========================
# Loop Through List
# ==========================

languages = ["Python", "JavaScript", "SQL"]

for language in languages:
    print(language)

# ==========================
# Nested Loop
# ==========================

for row in range(3):
    for col in range(3):
        print(f"({row}, {col})")

File: 02_javaScript/07_forLoop/forLoop.js
/*
-----------------------------------------
Topic : For Loop
Language : JavaScript
Bootcamp : Data Analytics Bootcamp
Author : Shubham Waghmare
-----------------------------------------
*/

// ==========================
// Basic for Loop
// ==========================

for (let i = 0; i < 5; i++) {
    console.log(i);
}

// ==========================
// Custom Range
// ==========================

for (let i = 1; i <= 5; i++) {
    console.log(i);
}

// ==========================
// Step Value
// ==========================

for (let i = 0; i <= 10; i += 2) {
    console.log(i);
}

// ==========================
// Reverse Loop
// ==========================

for (let i = 10; i >= 1; i--) {
    console.log(i);
}

// ==========================
// Loop Through String
// ==========================

let name = "Shubham";

for (let char of name) {
    console.log(char);
}

// ==========================
// Loop Through Array
// ==========================

let languages = ["Python", "JavaScript", "SQL"];

for (let language of languages) {
    console.log(language);
}

// ==========================
// Nested Loop
// ==========================

for (let row = 0; row < 3; row++) {
    for (let col = 0; col < 3; col++) {
        console.log(`(${row}, ${col})`);
    }
}